src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```

Comment:
Let's extract properties from the person here: 

`({ person: { age, name ....} })`